### DISCUSSION TITLE
Implement my own DBContext sqlserver  instead of yessql for users table

### DISCUSSION THREAD

**Discussion by webmedia1012** [OrchardCMS/OrchardCore#6906] [Q&A]
Hi folks,

Just wanted to get an understanding of how I can implement my own user, I know that OrchardCore already has identity management built in, but it looks up to OrchardCms database for users, I have my own user table and when I try to inject SignInManager<myownuserobj> and UserManager<myownuserobject> I get a "cannot resolve" error, I guess because I didn't wire them in the startup.cs thinking if I call services.AddOrchardCms it will take care of the wiring for me?  How do I go about this?

Currently I do this in the startup.cs
services.AddDbContext<appDbContext>(config=> { config.usesqlserver(configuration.GetConnectionString("myconn"));

then : serbbives.addOrchardCms();

Thank you hope this make sense.

Regards,
G

**Comment by hishamco** [comment]
I think it's the matter on when you inject your `SignInManager` and `UserManager` implementation

**Comment by sebastienros** [comment]
I assume you could create a module that provides custom implementations for UserManager and other identity services.
